app/importers/unpaywall_publication_importer.rb

```diff
@@ -110,4 +131,9 @@ def update_open_access_location(open_access_location, unpaywall_json)
     def build_new_oal(publication, url)
       publication.open_access_locations.build(source: Source::UNPAYWALL, url: url)
     end
+
+    def title_match?(title1, title2)
+      title1&.downcase&.gsub(/[^a-z0-9]/, '')&.include?(title2&.downcase&.gsub(/[^a-z0-9]/, '')) &&
+        title2&.downcase&.gsub(/[^a-z0-9]/, '')&.include?(title1&.downcase&.gsub(/[^a-z0-9]/, ''))
```

Comment:
Do we want to use `||` instead of `&&` here?  The way it would work right now I think would basically require and exact match.

Reply:
I did that to account for short titles that show up on some publications. Ex: 'Economic Development' matched for 'Economic Development, A Reflection', 'The Role of Agriculture in Economic Development', etc. Ideally we would fuzzy match, but from what I saw, it looked like we would need other gems to do so outside of a Postgres query

Reply:
So if we are checking that title 1 is included in title 2 _and_ title 2 is included in title 1, aren't we just doing an `==` operation?

Reply:
Yep. My bad. I can clean that up